runtime/caches/redis.ts

```diff
@@ -81,125 +89,78 @@ export const caches: CacheStorage = {
       ): Promise<ReadonlyArray<Request>> => {
         throw new Error("Not Implemented");
       },
-      /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Cache/match) */
+      delete: async (
+        request: RequestInfo | URL,
+        _?: CacheQueryOptions,
+      ): Promise<boolean> => {
+        console.debug("[cache]", "delete");
+
+        ensureConnection();
+
+        return requestURLSHA1(request)
+          .then(redis.del)
+          .then(() => true)
+          .catch((err: Error) => {
+            console.error(err);
```

Comment:
I think errors can be sent to hyperdx, import { logger } from "../../observability/mod.ts"